tests/loss/test_detcon_loss.py

```diff
@@ -0,0 +1,124 @@
+from math import e
+from typing import Any, List, Tuple
+from urllib import request
+from webbrowser import get
+
+import pytest
+import torch
+from torch import Tensor
+from torch import distributed as dist
+
+from lightly.loss import DetConBLoss, DetConSLoss
```

Comment:
There are a bunch of unused imports. I thought we remove them automatically with `make format` but this doesn't seem to be the case. Will have to enable that again.